Doc/library/inspect.rst

```diff
@@ -504,9 +504,9 @@ attributes (see :ref:`import-mod-attrs` for module attributes):
    are true.
 
    This, for example, is true of ``int.__add__``.  An object passing this test
-   has a :meth:`~object.__get__` method but not a :meth:`~object.__set__`
-   method, but beyond that the set of attributes varies.  A
-   :attr:`~definition.__name__` attribute is usually
+   has a :meth:`~object.__get__` method, but not a :meth:`~object.__set__`
```

Comment:
Maybe this part of the text requires a `.. versionadded:: 3.14`, but I am not sure on that

Reply:
I believe `.. versionchanged:: .....` would be more approproate, as here we change the behavior of an existing entity, rather than add a new entity.

Also – here be come back to the question I asked at the beginning – shouldn't it be applied also to Python 3.13? (considering that it is still in the beta phase, here we have a bugfix, rather than a feature, even though it seems too disruptive to be backported to the 3.12 and earlier Python versions)

What do you think?

Reply:
Yes, `versionchanged` would be the right docs notation.

Reply:
The note on https://docs.python.org/3/reference/datamodel.html#object.__set__ makes it clear that this is a bugfix (adding `__delete__` is enough to make a descriptor a data descriptor), so backporting to 3.13 makes sense to me.

Reply:
OK, added. :)